### PR TITLE
Change proposals to RIF Identity Docs

### PR DESCRIPTION
**Pull request by miltonwg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
-    RIF Identity is the identity and reputation layer within the RIF ecosystem.
+
+##  RIF Identity documentation
+
+<p align="left">
+    RIF Identity is the identity and reputation layer within the RIF ecosystem.<br><br>
+    It is meant to allow users to easily control their digital identities to interact in decentralized economies while building a self-sovereign identity and reputation. This will enable people, especially those excluded from the traditional financial system, to participate in the decentralized digital economy of the future.<br>
+    The main goal of RIF Identity is to protect users’ personal data in a privacy respectful way, empowering them to manage who can access it and giving them full control of their reputation so they can use it to interact with multiple marketplaces and platforms with freedom to move from one to another without losing their track record, contacts and social value. <br><br>
+    For the full documentation please go to <a href="https://rsksmart.github.io/rif-identity-docs/">RIF Identity Docs site</a>
 </p>
```
Comment (ilanolkies) [inline README.md:12]: I would move this bellow the title, without changing the header format.

File: README.md [hunk 2]
```diff
@@ -1,11 +1,16 @@
-<p align="middle">
+<p align="left">
```
Comment (ilanolkies) [inline README.md:1]:
```suggestion
<p align="middle">
```

:)

File: ssi/specs/convey-service.markdown [hunk 3]
```diff
@@ -6,9 +6,9 @@ layout: default
 
 A very common functionality nowadays is to allow the user to present their credentials using QR codes. Verifiable credentials can use much space and make the code unable to be displayed on phone screens or scanned by cameras. With that motivation, this protocol is designed to allow transporting a message between two parties, with a third party in the form of a relayer.
 
-Alice: wants to share a file with Bob showing a QR code
-Bob: wants to read the file using a QR code scanner
-Convey service: cachés files and publish files into IPFS
+Alice: wants to share a file with Bob showing a QR code  
+Bob: wants to read the file using a QR code scanner  
+Convey service: cachés and publishes files into IPFS
```
Comment (ilanolkies) [inline ssi/specs/convey-service.markdown:11]:
```suggestion
Alice: wants to share a file with Bob showing a QR code
Bob: wants to read the file using a QR code scanner
Convey service: cachés and publishes files into IPFS
```

File: ssi/specs/credential-requests.markdown [hunk 4]
```diff
@@ -2,11 +2,11 @@
 layout: default
 ---
 
-## Credential requests protocol - a simple protocol for request, grant and receive credential case
+## Verifiable Credential requests protocol - a simple protocol for request, grant and receive verifiable credential case
```
Comment (ilanolkies) [inline ssi/specs/credential-requests.markdown:5]: _Verifiable credential_ or _Verificable Credential_?

File: ssi/specs/credential-requests.markdown [hunk 5]
```diff
@@ -2,11 +2,11 @@
...
 
-This protocol is held by two entities: the _issuer_ and the _holder_. The issuer is a public entity that issues a specific type of credential, the holder is a user who wishes to acquire that credential. The issuer may require the holder to share specific information. This information can be either data input by the user in a declarative way (declarative details) or other credentials issued by the same issuer or another entity.
+This protocol is held by two entities: the _issuer_ and the _holder_. The issuer is a public entity that issues a specific type of credential, the holder is a user who wishes to acquire that credential and controls the private key associated to a DID. The issuer may require the holder to share specific information. This information can be either data input by the user in a declarative way (declarative details) or other credentials issued by the same issuer or another entity.
```
Comment (ilanolkies) [inline ssi/specs/credential-requests.markdown:9]: Is that necessary? What if he is delegated or was transferred the ownership of an identity. I would say "controls the identity".

File: ssi/specs/credential-requests.markdown [hunk 6]
```diff
@@ -29,6 +29,8 @@ The schema that is used to share credentials and declarative details in the cred
 
 - Support DID Document service detection - allow the holder to find the issuer's service endpoint using its DID Document
 - Provide a standard API to be implemented by holders
+- Schema discoverability methods
```
Comment (ilanolkies) [inline ssi/specs/credential-requests.markdown:32]:
```suggestion
- Standard method for schema discovery
```

**Review by ilanolkies — CHANGES_REQUESTED** [review]
LGTM

File: ssi/specs/convey-service.markdown (hunk 3 above)
Comment (miltonwg) [reply ssi/specs/convey-service.markdown:11]: @ilanolkies I added the 2 spaces for the preview to take the EOL.
If not the preview is like this:

image


https://github.com/rsksmart/rif-identity-docs/blob/master/ssi/specs/convey-service.markdown

File: ssi/specs/credential-requests.markdown (hunk 5 above)
Comment (miltonwg) [reply ssi/specs/credential-requests.markdown:9]: @ilanolkies 
If he has a delegated or transferred identity, in both cases he would also be controlling the private key associated to a DID.
In which cases he wouldn't be controlling a private key associated to a DID?

I want to emphasize this to explain it as simple as possible. The concept "control an identity" in the end is controlling the private keys associated to a DID.

What do you think?

File: README.md (hunk 1 above)
Comment (miltonwg) [reply README.md]: Changes commited!

File: ssi/specs/credential-requests.markdown (hunk 5 above)
Comment (ilanolkies) [reply ssi/specs/credential-requests.markdown:9]: What about

```suggestion
This protocol is held by two entities: the _issuer_ and the _holder_. The issuer is a public entity that issues a specific type of credential, the holder is a user who wishes to acquire that credential and controls a certain DID. The issuer may require the holder to share specific information. This information can be either data input by the user in a declarative way (declarative details) or other credentials issued by the same issuer or another entity.
```

**Review by ilanolkies — APPROVED** [review]
You rock 🤟🏻

**Pull request merged**